### ISSUE TITLE
Allow disabling linting, but not formatting

### ISSUE THREAD

**Issue by macintacos** [rvben/rumdl#677]
### Problem or motivation

Maybe this is already possible and I haven't found it, so apologies if that's the case.

Consider the following text:

```md
asdflkajsdl kfjhaksdlj fhalkjsdhf akljsdhf klajsdhk akdsjfha sldkjfh alksdjfh aklsdjfh alksdjfh alkjsdhf aklsdjhf laksdjf alkjsdf adkshf `a slkdjfhal ksdhflka sdhf alksdjh falksdjhf alksjdhf alksdj fhalksdj falksdj flaksdjh flaksdjh flaksdjf` asdfhlkajs dhflkajd shflkajsdf lakjsdf klajdsf klasdj flaksjd flakjdsf lakjsdfh laksdjf lkjh
```

With the following config:

```toml
[MD013]
# Reflow holds prose to the line length, but code/digraph blocks can't be wrapped
# to fit — exempt them so they don't fail an otherwise-clean file.
line-length = 90
reflow      = true
reflow-mode = "normalize"
```

Running `rumdl fmt` on the aforementioned file gives you the following:

```md
asdflkajsdl kfjhaksdlj fhalkjsdhf akljsdhf klajsdhk akdsjfha sldkjfh alksdjfh aklsdjfh
alksdjfh alkjsdhf aklsdjhf laksdjf alkjsdf adkshf
`a slkdjfhal ksdhflka sdhf alksdjh falksdjhf alksjdhf alksdj fhalksdj falksdj flaksdjh flaksdjh flaksdjf`
asdfhlkajs dhflkajd shflkajsdf lakjsdf klajdsf klasdj flaksjd flakjdsf lakjsdfh laksdjf
lkjh
```

This is fine, however, there's a catch - the long inline code actually still trips the line length lint check, so you still get the following warning:

```txt
TEST.md:3:91: [MD013] Line length 105 exceeds 90 characters
```

I have some paragraphs that contain long inline code incantations, and for uninteresting reasons it would be very clumsy to have these put into "proper" standalone code blocks, and they are frequent enough that I don't want to pepper the project with `rumdl-disable-next-line` everywhere, and I really would like to not have to disable the rule entirely on the whole file (since that would mean that reflowing would stop happening, which I generally want). Right now I'm basically stuck if I want to set a line length and have `reflow` turned on - these long inline code snippets can't be "broken up", and because of that, `rumdl` reports that the lines are too long.

### Proposed solution

Add some kind of option / directive to allow for disabling _just_ the "issues" that are found for a specific rule, but allow users to continue to have `rumdl fmt` reflow the text in those files. That would be the best of both worlds for me - I'd be able to ensure that most of everything that I want gets reflowed nicely, and the one-off scenarios in my docs that have a lot of long inline code can continue to exist without showing up as a lint problem.

### Feature category

CLI / configuration

### Example markdown

```markdown

```

### Contribution

- [ ] I would be willing to submit a PR to implement this feature

**Comment by rvben** [comment]
Thanks, @macintacos! Addressed in v0.2.19 for the case you described.

Rather than a general "lint but don't format" switch, we added a targeted MD013 option, `code-spans`. With `code-spans = false`, a line is exempt from the length warning when it would fit once non-breakable inline code spans are excluded. So `rumdl fmt` still reflows the surrounding prose, and `rumdl check` stops flagging a line whose only over-length content is an unbreakable backtick expression, which was the specific friction you hit:

```toml
[MD013]
code-spans = false
```

Closing since this resolves the reported problem. If you have a broader need for a general per-rule "format but don't lint" mode across other rules, please open a separate issue and we'll weigh it on its own.
